PeerioTechnologies/peerio-mobile
# feat: mobile e2e tests for medcryptor patient space

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/9476/mobile-e2e-tests-for-medcryptor-patient-space-navigation
#### Testing instructions  
To test local:
`npm run test-ios`
`npm run test-android`

Have also added a script to build the app and test all Medcryptor scenarios:
`scripts/cucumber-medcryptor.sh`
(they should have the tag `@medcryptor`)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
